app/controllers/api/v1/admin/roles_controller.rb

```diff
@@ -63,6 +63,9 @@ def update
         # Returns: { data: Array[serializable objects] , errors: Array[String] }
         # Does: Deletes a role.
         def destroy
+          default_roles = @role.name == 'User' || @role.name == 'Administrator' || @role.name == 'Guest'
```

Comment:
We can clean this up slightly - `['User','Administrator','Guest'].includes?(@role.name)`